src/index.js

```diff
@@ -6,8 +6,6 @@ const STEAM_DIR = process.env.STEAM_DIR;
 const STEAM_CMD = process.env.STEAM_CMD;
 const STEAM_TEMP = process.env.STEAM_TEMP;
 
-const IS_POST = !!core.getState('isPost');
```

Comment:
What do you think about writing these actions in typescript?

Reply:
Typescript is Javascript just with extra steps.

I just find that the additional files are more cumbersome to work with.